src/branch.rs

```diff
@@ -28,6 +28,16 @@ impl<'repo> Branch<'repo> {
         Branch { inner: reference }
     }
 
+    /// Ensure the branch name is well-formed.
+    pub fn name_is_valid(name: &str) -> bool {
+        let name = CString::new(name).unwrap();
+        unsafe { let mut valid: libc::c_int = 0;
```

Comment:
I am pretty sure the formatting is off. please run `cargo fmt`

Reply:
thanks!